threesixty/views.py

```diff
@@ -260,21 +262,39 @@ def post(self, request, *args, **kwargs):
 
     def get(self, request, *args, **kwargs):
         self.participant = get_object_or_404(models.Participant, email=self.email, survey=self.survey)
+        if self.kwargs.get('question_pk'):
```

Comment:
Honestly I wouldn't try to solve two different things in one view.
I would suggest to use a regular `UpdateView` when you want to go to a specify question.

Reply:
Would be nice if we can talk more in detail about this. I am not sure how to achieve that. As far as I can imagine it would mean there is another form that would post to another view when we want to undo?